packages/rule-components/src/ReportDetails/index.js

```diff
@@ -38,7 +39,7 @@ const ReportDetails = (
                     </div>
                 </StackItem>
                 <StackItem>
-                    <RuleFeedback ruleId={ ruleId }/>
+                    <RuleFeedback ruleId={ ruleId } onFeedbackChanged={ onFeedbackChanged }/>
```

Comment:
I think it will be nice to hide this item in case if `onFeedbackChanged ` is `undefined`